db/migrate/20171110171050_add_subscription_content.rb

```diff
@@ -0,0 +1,10 @@
+class AddSubscriptionContent < ActiveRecord::Migration[5.1]
+  def change
+    create_table :subscription_contents do |t|
+      t.references :subscription, null: false, foreign_key: true
+      t.references :notification, null: false, foreign_key: true
+      t.references :email, null: true, foreign_key: true
```

Comment:
Are we considering at this point archiving emails? as we might not want this to be an actual foreign key for that?

Haven't thought it through but wondering if we want cascades on other columns at all?

Reply:
I don't really mind either way re. foreign keys. We can remove them later if there's a need to archive this data, perhaps?

Which cascades would you want to add? Do you mean in the context of this change or are you thinking more generally about all the models in the system so far?

Reply:
Yeah if they're situations that aren't too problematic to swap up out later fine to defer until we know more.

I was just referring to this change here. Without any cascades any attempts to remove notification or subscription will fail. Is that cool?

Reply:
I think that's probably fine for now since we have no code to delete stuff, and it's a useful check to prevent accidental deletion in the console. We can make a decision on cascades when we need to start looking at how we can archive/delete stuff easily I think.

Reply:
Sure, sounds fair